src/framework/MOM_intrinsic_functions.F90

```diff
@@ -108,6 +203,12 @@ logical function intrinsic_functions_unit_tests(verbose)
   fail = fail .or. Test_cuberoot(v, 0.125)
   fail = fail .or. Test_cuberoot(v, 0.965)
 
+  testval = 1.0e-99
```

Comment:
Please add the following line in the list above:
`fail = fail .or. Test_cuberoot(v, 1.0 - epsilon(1.0))`

Reply:
Great idea, I've added the test and it appears to have passed.  On my machine, `1.0 - epsilon(1.0)` matches the quad precision value, including the ULP bit.